CONTRIBUTING.md

```diff
@@ -1,73 +1,70 @@
-# Contributing to Apps CLI plugin
+# Contributing to Apps Plugin for the Tanzu CLI
 
-The Apps CLI plugin project team welcomes contributions from the community. If you wish to contribute code and you have not signed our contributor license agreement (CLA), our bot will update the issue when you open a Pull Request. For any questions about the CLA process, please refer to our [FAQ](https://cla.vmware.com/faq).
+We welcome contributions from the community and first want to thank you for taking the time to contribute!
 
-## Contribution Flow
+Please familiarize yourself with the [Code of Conduct](https://github.com/vmware/.github/blob/main/CODE_OF_CONDUCT.md) before contributing.
 
-This is a rough outline of what a contributor's workflow looks like:
+* _CLA: Before you start working with Apps Plugin for the Tanzu CLI, please read and sign our Contributor License Agreement [CLA](https://cla.vmware.com/cla/1/preview). If you wish to contribute code and you have not signed our contributor license agreement (CLA), our bot will update the issue when you open a Pull Request. For any questions about the CLA process, please refer to our [FAQ]([https://cla.vmware.com/faq](https://cla.vmware.com/faq))._
 
-- Create a topic branch from where you want to base your work
-- Make commits of logical units
-- Make sure your commit messages are in the proper format (see below)
-- Push your changes to a topic branch in your fork of the repository
-- Submit a pull request
+## Ways to contribute
 
-Example:
+We welcome many different types of contributions and not all of them need a Pull request. Contributions may include:
 
-``` shell
-git remote add upstream https://github.com/vmware-tanzu/apps-cli-plugin.git
-git checkout -b my-new-feature main
-git commit -a
-git push origin my-new-feature
-```
+* New features and proposals
+* Documentation
+* Bug fixes
+* Issue Triage
+* Answering questions and giving feedback
+* Helping to onboard new contributors
+* Other related activities
 
-### Staying In Sync With Upstream
+## Getting started
 
-When your branch gets out of sync with the vmware-tanzu/main branch, use the following to update:
+### Development Environment Setup
 
-``` shell
-git checkout my-new-feature
-git fetch -a
-git pull --rebase upstream main
-git push --force-with-lease origin my-new-feature
-```
+Follow the documentation in the development document for the [pre-requisites](./Development.md#Prerequisites) required for the setup, as well [building](./Development.md#building) the project.
```

Comment:
pre-reqs and building links should be capital `./DEVELOPMENT.md`

Reply:
This still seems to be pointing to a page that does not exists.

`./Development.md#Prerequisites` should be `./DEVELOPMENT.md#Prerequisites`
`./Development.md#building` should be `./DEVELOPMENT.md#building`

Reply:
Good to know. Surprisingly in Visual studio code, links worked. Looks like github links is strongly typed. Learnt something new :)